src/uiveri5.js

```diff
@@ -270,31 +270,51 @@ function run(config) {
 
       });
 
-      // override with added logging and parameter manipulation
+      // used in protractor for many scripts and notably for waitForAngular
+      // override with added logging
       var origExecuteAsyncScript_= browser.executeAsyncScript_;
       browser.executeAsyncScript_ = function() {
+        var code = arguments[0];
+        var scriptName = arguments[1];
+        var params = arguments[2];
         // log script execution
-        logger.trace('Execute async script: ${name}, code:\n ${JSON.stringify(code)}',
-          {name:  arguments[1], code: arguments[0]});
-        // override the timeout used by waitForAngular
-        arguments[2] = JSON.stringify({
-          waitForUI5Timeout: waitForUI5Timeout
```

Comment:
isn't the timeout needed?

Reply:
this code seems to be a leftover from some previous incarnation, currently the timeout is set in loading of the waiter.